lms/djangoapps/teams/static/teams/templates/team-actions.underscore

```diff
@@ -1,4 +1,4 @@
 <div class="team-actions">
   <h3 class="title"><%- gettext("Are you having trouble finding a team to join?") %></h3>
-  <p class="copy"><%= message %></p>
-</div>
\ No newline at end of file
+  <p class="copy"><%= HtmlUtils.escape(messageHtml) %></p>
```

Comment:
Given that messageHtml is now HTML (and presumably properly escaped as it was built up), why does it need to be escaped here again? I'm not understanding the complete workflow.